index.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/getNyumba', {useNewUrlParser: true});
+const db = mongoose.connection();
+
+db.on('error', console.error.bind('Connection error:'));
+
+const app = express();
+
+app.get('/', (req, res) => {
+    res.send('Hello House Hunters!')
+})
+
+app.listen(8000, () => {
```

Comment:
U can make the port to be read from the env variables since it most cases it will be defined by the server we shall host the application on 
`const port = process.env.PORT || 8000;`